### PR TITLE
Make sass command compatible with macos

### PR DESCRIPTION
**Pull request by marcoambrosini**
Use `printf` instead of `echo -n`

Screenshot 2024-03-25 at 12 59 55

### CODE REVIEW

**Review by ChristophWurst — APPROVED** [review]

**Comment by ChristophWurst** [comment]
Suggested commit title: `chore: Make sass command compatible with macos`

https://www.conventionalcommits.org/en/v1.0.0/#summary

File: package.json [hunk 1]
```diff
@@ -12,14 +12,14 @@
     "postbuild": "npm run sass && npm run sass:icons",
     "dev": "webpack --node-env development --progress",
     "watch": "webpack --node-env development --progress --watch",
-    "lint": "eslint $(for appdir in $(ls apps); do if ! $(git check-ignore -q $appdir); then echo -n \"$appdir \"; fi; done) core --no-error-on-unmatched-pattern",
-    "lint:fix": "eslint $(for appdir in $(ls apps); do if ! $(git check-ignore -q $appdir); then echo -n \"$appdir \"; fi; done) core --no-error-on-unmatched-pattern --fix",
+    "lint": "eslint $(for appdir in $(ls apps); do if ! $(git check-ignore -q $appdir); then printf \"$appdir \"; fi; done) core --no-error-on-unmatched-pattern",
+    "lint:fix": "eslint $(for appdir in $(ls apps); do if ! $(git check-ignore -q $appdir); then printf \"$appdir \"; fi; done) core --no-error-on-unmatched-pattern --fix",
```
Comment (Altahrim) [inline package.json:16]: I am not sure the `-n` if even necessary: 
```suggestion
    "lint:fix": "eslint $(for appdir in $(ls apps); do if ! $(git check-ignore -q $appdir); then echo $appdir; fi; done) core --no-error-on-unmatched-pattern --fix",
```

**Review by Altahrim — APPROVED** [review]

**Review by emoral435 — APPROVED** [review]
Either removing the -n or using printf works :) nice change!

File: package.json (hunk 1 above)
Comment (giantmolecules1) [reply package.json:16]: Can't

**Pull request merged**